src/module/actor/loot/sheet.ts

```diff
@@ -84,6 +86,65 @@ export class LootSheetPF2e<TActor extends LootPF2e> extends ActorSheetPF2e<TActo
         data.hidden = isMerchant && item.isOfType("treasure") && item.isCoinage && !item.container;
         return data;
     }
+
+    private async transferToPartyStash(): Promise<void> {
+        const activeParty = game.actors.party;
+
+        if (!activeParty) {
+            ui.notifications.warn("No active party found.");
```

Comment:
Instead of sending a notification, my preference is to disable the button (using `{{disabled hasActiveParty}}` in sheet), and then early exit using `if (!activeParty) return;`.

Reply:
@CarlosFdez fair point, done

Reply:
@CarlosFdez is the accepted practice here for me to mark these items resolved or you/the reviewer?